src/index.js

```diff
@@ -912,7 +926,7 @@ class MiniCssExtractPlugin {
                       "}",
                     ]),
                 "return linkTag;",
-              ]
+              ].filter((statement) => statement !== undefined)
```

Comment:
Why?

Reply:
> Why?

Thank you for your reply

Because the following code may return undefined

```js
MiniCssExtractPlugin.getHooks(compiler).beforeTagInsert?.call({
  varNames: {
    chunkId: "chunkId",
    tag: "linkTag",
  },
})
```

if I set this to an empty string, it will cause the test case to time out, and I haven't found a way to update the test case code (other than manually)

```js
MiniCssExtractPlugin.getHooks(compiler).beforeTagInsert?.call({
  varNames: {
    chunkId: "chunkId",
    tag: "linkTag",
  },
}) || "",
```